tests/main/hooks-env/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Check that environment variables are interpolated when running hooks.
+
+prepare: |
+    snapbuild $TESTSLIB/snaps/snapctl-hooks .
+    snap install --dangerous snapctl-hooks_1.0_all.snap
+
+restore: |
+    rm snapctl-hooks_1.0_all.snap
```

Comment:
You probably don't need this if you use one of the snap build helpers. They also cache the build across tests.

Reply:
Good point, I keep forgetting about the build helper. Done.